docusaurus/docs/develop/path/4_path_localnet.md

```diff
@@ -0,0 +1,347 @@
+
+#### `make localnet_k9s`
+
+Launches [k9s](https://k9scli.io/), a terminal-based Kubernetes UI, inside the container.
+
+```bash
+make localnet_k9s
+```
+
+k9s provides an interactive way to:
+
+- Navigate Kubernetes resources
+- View and follow logs
+- Execute into pods
+- Edit resources
+- Monitor resource usage
+
+![k9s Dashboard](../../../static/img/k9s-localnet.png)
+_k9s running inside the PATH localnet Docker container_
+
+:::tip k9s Quick Commands
+
+- `:pods` - List all pods
+- `:svc` - List all services
+- `l` - View logs for selected resource
+- `d` - Describe selected resource
+- `s` - Shell into selected pod
+- `ctrl+a` - Show all namespaces
+- `?` - Show help menu
+- :::
+
+#### `make localnet_exec`
+
+Opens an interactive shell inside the running PATH Localnet container.
+
+```bash
+make localnet_exec
+```
+
+Useful for:
+
+- Running kubectl commands directly
+- Inspecting logs and configurations
+- Debugging networking issues
+- Managing the Kind cluster
+
+Example session:
+
+```bash
+$ make localnet_exec
+root@path-localnet:/app# kubectl get pods
+NAME                           READY   STATUS    RESTARTS   AGE
+path-5f7b9c4d6f-abc12         1/1     Running   0          5m
+envoy-gateway-xyz789          1/1     Running   0          5m
+grafana-6d8f9c7b5-def45      1/1     Running   0          5m
+
+root@path-localnet:/app# kubectl logs path-5f7b9c4d6f-abc12
+```
+
+## Container Environment
+
+The PATH Localnet image includes all necessary development tools, meaning you can run PATH, GUARD, and WATCH locally without any additional dependencies.
+
+- Image: `ghcr.io/buildwithgrove/path-localnet-env`
+- [GHCR Repository](https://github.com/orgs/buildwithgrove/packages/container/package/path-localnet-env)
+
+### Installed Tools
+
+- **Go**: For building PATH
+- **Tilt**: Development orchestrator
+- **kubectl**: Kubernetes CLI
+- **Kind**: Kubernetes in Docker
+- **Helm 3**: Package manager for Kubernetes
+- **k9s**: Terminal UI for Kubernetes
+- **Docker**: Docker-in-Docker for Kind cluster
+- **ajv-cli**: For validating configuration schema
+- **yq/jq**: YAML/JSON processors
+
+### File Mounts
+
+The container mounts your local PATH repository at `/app`, enabling:
+
+- Hot reloading of Go code changes
+- Configuration file updates
+- Access to test data and scripts
+
+### Configuration Validation
+
+On startup, the container validates your `./local/.config.yaml` against the [YAML schema](https://github.com/buildwithgrove/path/blob/main/config/config.schema.yaml) in the PATH repo.
+
+## Development Workflow
+
+### Hot Reloading
+
+The PATH Localnet supports hot reloading for rapid development:
+
+1. **Make code changes** in your local PATH repository
+2. **Save the file**
+3. **Tilt detects changes** and triggers a rebuild
+4. **New binary is deployed** to the Kind cluster
+5. **Service restarts** with your changes
+
+### Viewing Logs
+
+Multiple ways to view logs:
+
+1. Recommended: **Tilt UI** (http://localhost:10350):
```

Comment:
Highly recommend using subsections instead of numbered lists for this.

Easier to format, less indenting and you can link to it!